deployment/scripts/fuel-core-deploy.sh

```diff
@@ -23,10 +23,6 @@ if [ "${k8s_provider}" == "eks" ]; then
               --wait \
               --timeout 8000s \
               --debug
-    
-    # Verify Deployment
-    kubectl rollout status deployment/fuel-core-k8s --namespace ${k8s_namespace} --timeout=120s
```

Comment:
why are we removing this? Is this wait already handled by helm?

Reply:
helm chart deployment won't successfully complete/finish unless the pods are up and running - this command is extraneous and not needed

if the pods are not up and running - helm will keep waiting till pods are ready and in event, there is an error, it will go based on the 8000 second timeout

this command is extraneous